cellpack/bin/pack.py

```diff
@@ -70,6 +70,9 @@ def pack(recipe, config_path=None, analysis_config_path=None):
             recipe_version=recipe_data["version"],
             image_export_options=packing_config_data.get("image_export_options"),
             parallel=packing_config_data.get("parallel", False),
+            save_gradient_data_as_image=packing_config_data.get(
+                "save_gradient_data_as_image", False
+            ),
```

Comment:
this is starting to make me think we should just sent the config into analysis instead of unpacking values here

Reply:
Created an issue #227 to track this